### PR TITLE
[BUGFIX] Fix alignment 💯

### PR DESCRIPTION
**Pull request by DominikBucher12**
# What this PR do:
- Fixes text alignment setting
# How to test it
- Run app and see for yourself.

https://github.com/danielsaidi/RichTextKit/assets/17381941/60f21e75-26f8-440e-b73f-38523ba7b268

### CODE REVIEW

File: Sources/RichTextKit/RichTextView_UIKit.swift [hunk 1]
```diff
@@ -101,6 +101,12 @@ open class RichTextView: UITextView, RichTextViewComponent {
     /// Keeps track of the data format used by the view.
     private var richTextDataFormat: RichTextDataFormat = .archivedData
 
+    /// Get the text alignment.
+    public var richTextAlignment: RichTextAlignment?
+    //    {
+    //        guard let style = richTextParagraphStyle else { return nil }
+    //        return RichTextAlignment(style.alignment)
+    //    }
```
Comment (DominikBucher12) [inline Sources/RichTextKit/RichTextView_UIKit.swift:109]: Will delete this when e are aligned :)

File: Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift [hunk 2]
```diff
@@ -17,10 +17,11 @@ import AppKit
...
     /// Get the text alignment.
-    var richTextAlignment: RichTextAlignment? {
-        guard let style = richTextParagraphStyle else { return nil }
-        return RichTextAlignment(style.alignment)
-    }
+//    var richTextAlignment: RichTextAlignment? 
+//    {
+//        guard let style = richTextParagraphStyle else { return nil }
+//        return RichTextAlignment(style.alignment)
+//    }
```
Comment (DominikBucher12) [inline Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift:24]: same here.

File: Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift [hunk 3]
```diff
@@ -16,12 +16,6 @@ import AppKit
 
 public extension RichTextViewComponent {
 
-    /// Get the text alignment.
-    var richTextAlignment: RichTextAlignment? {
```
Comment (DominikBucher12) [inline Sources/RichTextKit/Component/RichTextViewComponent+Alignment.swift:20]: This needs stays. I overcomplicated this a bit... 😮‍💨

**Comment by danielsaidi** [comment]
@DominikBucher12 Nice video - seems to work great! I was under the impression that alignment works in the current `main` branch? What bugs does this PR fix?

**Comment by DominikBucher12** [comment]
> @DominikBucher12 Nice video - seems to work great! I was under the impression that alignment works in the current `main` branch? What bugs does this PR fix?

Hola 👋 

I think theres was issue when you had multiple different paragraphs and pressed newline on the line before that alignment went to left instantly but I cannot honestly remembered there were some little unconvenient issues :/

**Comment by DominikBucher12** [comment]
In favore of feature/RichTextKit2 which contains TextKit2 changes crucial for working bullet and numbered list, I am closing this.

**Pull request closed without merging**